fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/physical/PhysicalHashJoin.java

```diff
@@ -140,32 +138,32 @@ public <R, C> R accept(PlanVisitor<R, C> visitor, C context) {
         return visitor.visitPhysicalHashJoin(this, context);
     }
 
-    @Override
-    public String toString() {
-        List<Object> args = Lists.newArrayList("type", joinType,
```

Comment:
this functions is remove?

Reply:
yes, move to its parent class
LogicalHashJoin and physical hash join share the same toString() fucntion